### PR TITLE
Properties#Show - Instructions tab redesign

### PR DESCRIPTION
**Pull request by cmckee-dev**
Screenshot_2019-10-23_14-23-10

Depends on PR #719

### CODE REVIEW

File: app/views_redesigned/async_property_card_footers/show.html.erb [hunk 1]
```diff
@@ -45,4 +45,3 @@
     </div>
   <% end %>
 </div>
-
```
Comment (andrewmcodes) [inline app/views_redesigned/async_property_card_footers/show.html.erb]: Can you revert this change? We would like to always have a new line at the end of each file.

Comment (cmckee-dev) [reply app/views_redesigned/async_property_card_footers/show.html.erb]: Oops. Fixed.
